Development/src/pages/receivers.js

```diff
@@ -544,3 +578,199 @@ const EditStagedTab = props => (
         </SimpleForm>
     </Edit>
 );
+
+const ConnectionManagementTab = ({
+    controllerProps,
+    receiverData,
+    ...props
+}) => {
+    const [sendersListData, setSendersListData] = useState(undefined);
+    const [params, setParams] = useState({
+        filter: {},
+    });
+
+    useEffect(() => {
+        (async function fetchData() {
+            const data = await dataProvider('GET_LIST', 'senders', params);
+            setSendersListData(data);
+        })();
+    }, [params]);
+
+    const changeFilter = (filterValue, name) => {
+        let newFilter = params.filter;
+        if (filterValue) {
+            newFilter[name] = filterValue;
+        } else {
+            delete newFilter[name];
+        }
+        setParams({ filter: newFilter });
+    };
+
+    const nextPage = async label => {
+        const data = await dataProvider(label, 'senders');
+        setSendersListData(data);
+    };
+
+    // receiverData initialises undefined, update when no longer null
+    useEffect(() => {
+        if (receiverData !== null)
+            setParams({
```

Comment:
Does this _combine_ with any user-specified filter as we want, or would they replace each other?